Source/UI/NewUI/BillingForm/Style Protocol/BillingFormStyle.swift

```diff
@@ -0,0 +1,7 @@
+import UIKit
+
+public protocol BillingFormStyle {
```

Comment:
Separate question : do we need to mark public methods  with prefix @objc for merchants using objc in their app.

Reply:
I will addd a new ticket for Objective C integration